### DISCUSSION TITLE
Certain letters not working sometimes.

### DISCUSSION THREAD

**Discussion by PheonixAi420** [lllyasviel/Fooocus#3091] [Q&A] [answered]
Anyone else having any issues with some letters not registering when you type? My keyboard works fine on everything else but sometimes the R S and F  letters doesn't seem to respond.

**Comment by mashb1t** [comment]
Fooocus uses html input elements as all other websites also do, nothing fancy. If your keyboard doesn't work here, it also doesn't work anywhere else, I'm sorry.

**Reply by PheonixAi420** [reply]
See that's where I'm confused. Brand new keyboard and sometimes a few letters don't work when typing prompts but when you switch tabs to say google, spotify etc. the keys work just fine. So I'm pretty sure my keyboard is not the problem. Works great on everything else. Seemed to start happening after the new update.

**Reply by mashb1t** [reply, marked as answer]
I've double-checked and it seems you are right, but only when hovering over the canvas with your mouse pointer, as they are shortcuts for actions on the canvas 🤯.

You can find the actions below:

![image](https://github.com/lllyasviel/Fooocus/assets/9307310/f1df86b9-2789-4023-8cab-e8ffbfa889b0)

Code:
https://github.com/lllyasviel/Fooocus/blob/05ef3e68619114492c11d4c4547df7b245520040/javascript/zoom.js#L32-L44

**Reply by PheonixAi420** [reply]
Ahh okay, yeah I knew it was something just wasn't sure what. Thanks.

**Reply by PheonixAi420** [reply]
Unfortunately, this is still happening when the mouse is not hovering over the canvas. It's a bug, it has to be. I've been having to copy and paste letters here and there to type the prompt correctly.

**Reply by mashb1t** [reply]
@CerberusAi420 Sadly i can''t reproduce this. Can you please create a step-by-step guide how to reproduce the issue or record your screen? Thanks a lot.

**Reply by PheonixAi420** [reply]
I will try the next time it happens. I'm not sure how to recreate it, it just happens randomly. I've been using fooocus all day and it only happens maybe 10-20% of the time I'm using it. It;s just kind of a pain when it does happen. I even moved my cursor off the screen and it still happened and I also noticed I would go to type something starting with the letter "F" and it would unselect the typing area for some reason. I'm not sure what is going on but I will try to help figure this out if I can. Thanks for the reply.

**Comment by PheonixAi420** [comment]
https://github.com/lllyasviel/Fooocus/assets/170890859/f90abef0-7d57-47d3-8d2b-f90f84a78ef2

Okay so in the first couple seconds I am trying to type "S" to spell ass and then at the end I'm typing "F" and it unselects the bow entirely. I don't know if this helps you or not, but I tried.

**Reply by mashb1t** [reply]
Can you please additionally check if this is related to a specific browser? Maybe this helps to reproduce the issue.

**Reply by PheonixAi420** [reply]
Im using Chrome. It automatically open in chrome. It really only started after the new update a few days ago.
